plugins/FakeDeaf/fakedeaf.plugin.js

```diff
@@ -122,6 +121,15 @@ module.exports = class FakeDeaf {
     this.fakeDeafEnabled = false
   }
 
+  infoType(statusBool) {
+    if (statusBool) {
+      return "success"
+    }
+    else {
+      return "default"
```

Comment:
Using `"default"` as a type leaves a blank space for an icon, but an empty string doesn't.

```suggestion
      return ""
```